src/openzaak/components/catalogi/admin/zaaktypen.py

```diff
@@ -307,6 +307,16 @@ def get_related_objects(self, obj):
 
         return resource_list, id_list
 
+    def response_post_save_change(self, request, obj):
+        if "_publish" in request.POST:
+            # publish related types
+            for besluittype in obj.besluittypen.filter(concept=True):
+                besluittype.publish()
```

Comment:
Add a message to state which objects have been auto-published, this to avoid surprises